news/move-cond

```diff
@@ -1,14 +0,0 @@
-**Added:**
-
-**Changed:**
-
-**Deprecated:**
-
-**Removed:**
-
-**Fixed:**
-
-* inconsistent handling of ctrl/meta + left and ctrl/meta + right
-  * ctrl+right used to be swapped with meta+right
-
-**Security:**
```

Comment:
Please don't remove this file

Reply:
Yes I can fix that.....no problem.  I thought I had to delete the file because it was in the last release.